pkg/registry/core/service/allocator/storage/storage.go

```diff
@@ -80,17 +80,12 @@ func NewEtcd(alloc allocator.Snapshottable, baseKey string, resource schema.Grou
 	}, nil
 }
 
-// Allocate attempts to allocate the item locally and then in etcd.
+// Allocate attempts to allocate the item.
 func (e *Etcd) Allocate(offset int) (bool, error) {
 	e.lock.Lock()
 	defer e.lock.Unlock()
 
-	ok, err := e.alloc.Allocate(offset)
```

Comment:
This looks like one of those "how did it ever work" cases.  I tried tracing thru code, but found it incredibly tedious to back-track thru interfaces.

I am going to do some digging, but this SEEMS correct...  back in a while.